### PR TITLE
fixed mispelling on the connections page

### LINKED ISSUE #8612: [UI] Misspelling in connections page
### Current Behavior
There is misspelling on the connections page
image
It shows Server builsd SHA:
### Desired Behavior
Misspell fixed.

### Implementation

Commit changes, push to the fork, create a PR.

### Mockups

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by nickvo08**
**Notes for Reviewers**

This PR fixes #8612

This PR fixes UI Misspelling in connections page #8612

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@vobanghia12 Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Review by AugustasV — CHANGES_REQUESTED** [review]

File: ui/components/Dashboard/ConnectionCharts.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-    getConnectionStatusSummary().then(json => {
-      setChartData(json.connections_status
-        .filter(data => isValidColumnName(data.status))
-        .map(data => [data.status,data.count]))
-    })
-  },[])
+    getConnectionStatusSummary().then((json) => {
+      setChartData(
+        json.connections_status
+          .filter((data) => isValidColumnName(data.status))
```
[the remark this answers is not in the document]
Comment (vobanghia12) [reply ui/components/Dashboard/ConnectionCharts.js:17]: It's just because of prettier extension

[the remark this answers is not in the document]
Comment (Ckbhatia) [reply ui/components/Dashboard/ConnectionCharts.js:17]: You need to undo this change if it's due to your personal preference settings. It's totally unnecessary until it was badly formatted or not following project-level preferences.

[the remark this answers is not in the document]
Comment (vobanghia12) [reply ui/components/Dashboard/ConnectionCharts.js:17]: Ok thanks I got it

[the remark this answers is not in the document]
Comment (leecalcote) [reply ui/components/Dashboard/ConnectionCharts.js:17]: @vobanghia12 will you be correcting this in the current PR or closing and opening a new one?

@Ckbhatia thank you for your reviews. Most helpful. 💯

[the remark this answers is not in the document]
Comment (vobanghia12) [reply ui/components/Dashboard/ConnectionCharts.js:17]: @leecalcote yes I closed it thanks

**Pull request closed without merging**